src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -38,7 +38,7 @@
 /**
  * Handles fetching and saving {@link Message} instances.
  */
-        
+
```

Comment:
Did you inadvertently add a new line here? Revert this change.

Reply:
Yes. 
It has been corrected.